authors/apps/authentication/tests/test_models.py

```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase
+
+from ..models import User
+
+class UserModelTest(TestCase):
```

Comment:
Always end the test classes with the word `TestCase`
```suggestion
class UserModelTestCase(TestCase):
```